src/ck-ldb/LBManager.h

```diff
@@ -34,14 +36,13 @@ class CkLBArgs
   int _lb_statson;              // stats collection
   int _lb_traceComm;            // stats collection for comm
   int _lb_central_pe;           // processor number for centralized startegy
-  int _lb_percentMovesAllowed;  // Specifies restriction on num of chares to be moved(as a
-                                // percentage of total number of chares). Used by RefineKLB
-  int _lb_teamSize;        // specifies the team size for TeamLB
-  int _lb_maxDistPhases;   // Specifies the max number of LB phases in DistributedLB
-  double _lb_targetRatio;  // Specifies the target load ratio for LBs that aim for a
-                           // particular load ratio
+  int _lb_teamSize;		// specifies the team size for TeamLB
+  int _lb_maxDistPhases;  // Specifies the max number of LB phases in DistributedLB
```

Comment:
Maybe not relevant for this patch, but it might be better to have each LB define it's own options. Otherwise this list is just going to get bloated.